resources/assets/styles/layout.less

```diff
@@ -58,7 +60,7 @@ header {
     }
 
     .dropdown-menu {
-      left: auto;
+      left: auto !important;
```

Comment:
angular-strap adds style on the element which overwrites left so I need the !important unless there's a toggle in angular-strap. Never worked with that library before :/